docs/guides/Checkout/configuration-section-api-quick-start-guides/get-and-update-the-seller-exchange-window.md

```diff
@@ -2,36 +2,35 @@
 ---
 
-[Marketplace](https://help.vtex.com/en/tutorial/integrating-with-marketplace) is the platform where products from different sellers can be offered to customers in one place. After the customer completes a purchase and the order is created, there may be scenarios in which the original seller of the product needs to be modified, for example, when their cancels the order due to an internal unavailability of product fulfillment.
+[Marketplace](https://help.vtex.com/en/tutorial/integrating-with-marketplace) is a platform where different sellers can sell their products in a single place. After the customer makes a purchase and the order is created, there may be scenarios in which the original seller of the product needs to be modified, for example, when they cancel the order due to internal unavailability of product fulfillment.
 
-In this situation, VTEX allows the marketplace to have a period of time (window), after the customer completes the purchase, to change sellers and thus avoid an order cancellation. For more information on the process of switching sellers, access the guides in the [Help Center](https://help.vtex.com/en/tutorial/change-seller--5TBAwO2kOAMw44uyaaQMQO) and the [Developer Portal](https://developers.vtex.com/vtex-rest-api/docs/change-seller).
+In such situations, VTEX allows the marketplace to have a period (window) after the customer makes the purchase to change sellers and avoid order cancellation. For more information on the process of changing sellers, please read the guides in the [Help Center](https://help.vtex.com/en/tutorial/change-seller--5TBAwO2kOAMw44uyaaQMQO) and the [Developer Portal](https://developers.vtex.com/vtex-rest-api/docs/change-seller).
+> ⚠️ The default window for changing a seller is <strong>2 days</strong>. However, you can change it for up to <strong>30 days</strong>.
 
->⚠️ The default window for the change seller is <strong>2 days</strong>. However, you can modify the value for up to a maximum period of <strong>30 days</strong>.
 
+This guide describes how to get a seller change window or update the number of days in a marketplace on your account.
 
-This guide describes how to get a seller's current exchange window or update this value in a marketplace on your account.
+## Getting the current seller change window
 
-## Getting the current window to change seller
-
-To get the window to change seller in a marketplace, you need to use the [Get window to change seller](https://developers.vtex.com/vtex-rest-api/reference/getwindowtochangeseller) endpoint as a **GET** request. In this request, you must send the `accountname` through the URL address, as shown by the example below:
+To get seller change window in a marketplace, you need to use the [Get window to change seller](https://developers.vtex.com/vtex-rest-api/reference/getwindowtochangeseller) endpoint as a **GET** request. In this request, you must send the `accountname` in the URL address, as in the example below:
```

Comment:
Atualizar link: [Get window to change seller](https://developers.vtex.com/docs/api-reference/checkout-api#get-/api/checkout/pvt/configuration/window-to-change-seller)